### ISSUE TITLE
Back button to navigate back from detailed pages

### ISSUE THREAD

**Issue by clundin55** [aniehuser/senior-design-group10#112]

**Comment by aniehuser** [comment]
- [ ] back button from the detailed run page

- [ ] back button from the detailed workflow page

**Comment by Brewer-Slack** [comment]
Is this necessary to include? We have a back button in the browser. This would make more sense to include if the page appeared as another layer on top of the current page with an x in the corner. We have two entirely different web pages however

**Comment by aniehuser** [comment]
@Brewer-Slack Yes it's necessary. I got several people giving feedback during usability testing asking for this functionality
